frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -163,19 +165,7 @@ const getCreateProps = ({ namespace }: { namespace: string }) => {
 
```

Comment:
you can also use the constants to create the object keys above

Reply:
done